spack/packages/dla-future/package.py

```diff
@@ -16,7 +16,10 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
     git = "https://github.com/eth-cscs/DLA-Future.git"
     maintainers = ["rasolca", "albestro", "msimberg", "aurianer"]
 
-    version("0.1.0", sha256="f7ffcde22edabb3dc24a624e2888f98829ee526da384cd752b2b271c731ca9b1")
+    version(
+        "0.1.0",
+        sha256="f7ffcde22edabb3dc24a624e2888f98829ee526da384cd752b2b271c731ca9b1",
+    )
```

Comment:
I applied `black` so some of the changes in `package.py` are un-related to the PR (and only consist in formatting).

Reply:
Fixed formatting by using `black -l 99 -C`, according to Spack's `pyproject.toml` settings.